Docs/API.md

```diff
@@ -781,6 +781,8 @@ __Parameters__
 | ``data``  | _Stream_  | Stream to upload |
 | ``size``  | _long_    | size of stream   |
 | ``contentType``  | _string_ | Content type of the file. Defaults to "application/octet-stream" |
+| ``metaData``  | _Dictionary<string,string>_ | Dictionary of metadata headers. Defaults to null. |
```

Comment:
The description for `metadata` could be,

Object metadata to be stored. Defaults to _null_.